### PR TITLE
See all reminders #1

### LINKED ISSUE #1: See All Reminders
When the user visits the root of the application (e.g. the "/" URL), they should see a list of all of the reminders on the page.

- Each reminder should have a class of .spec-reminder-item.

- There is a failing test set up in tests/acceptance/reminder-list-test.js.

Hot Tip: If you put “Closes #1” (assuming the issue is marked #1) in the pull request, your commit, or anywhere in the body of your PR, it will automatically close this issue when it successfully merged into master.

### PR DESCRIPTION
**Pull request by ccgamble**
## Purpose
@martensonbj @tman22 @stevekinney @nfosterky @nmchambs
This pull request renders the five reminders(title, date, and notes). They each have the given class name and are rendered in the root directory. After completing this tasks, in the reminder-list-test, it passes the 'viewing the homepage test' and fails the 'clicking on an individual item' test. Closes #1 

## Approach

This change allows the reminders to be rendered to the page.

### Learning

We referenced the repos from class to see the best way to setup and render. We also looked at the Mirage docs.

### Open Questions and Pre-Merge TODOs

- [x] Each reminder should have a class of .spec-reminder-item
- [x] Render the five reminders from store
- [x] There is a failing test set up in tests/acceptance/reminder-list-item-test.js
- [x] When the user visits the root of the application, they should see a list of all of the reminders on the page

### Test coverage 

No tests were written for this issue.

### Merge Dependencies and Related Work

N/A

### Follow-up tasks

TBD

#### After

### CODE REVIEW

File: app/models/reminder.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+	title: DS.attr('string'),
+	date: DS.attr('string'),
```
Comment (nfosterky) [inline app/models/reminder.js]: The date attribute should not be a string. Check out the docs here https://guides.emberjs.com/v2.10.0/models/defining-models/#toc_transforms

File: app/templates/reminders.hbs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+	<ul>
+	{{#each model as |reminder|}}
+	<li class="spec-reminder-item">
+		<p>{{reminder.title}}</p>
```
Comment (nfosterky) [inline app/templates/reminders.hbs]: Use a more semantic html element for titles / headers

File: app/routes/reminders.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+export default Ember.Route.extend({
+	model: function() {
+		return this.store.findAll('reminder');
```
Comment (nfosterky) [inline app/routes/reminders.js]: this.get('store') is a safer, long-term solution to retrieving info from the store

**Comment by ccgamble** [comment]
Thanks for the feedback. I made those changes and pushed them up to this PR.

@martensonbj @tman22 @stevekinney @nfosterky

File: app/routes/reminders.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+export default Ember.Route.extend({
```
Comment (stevekinney) [inline app/routes/reminders.js]: Let's add a new line here.

File: app/routes/reminders.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+	model: function() {
```
Comment (stevekinney) [inline app/routes/reminders.js]: Use the ES6 Object literal syntax here. Notice the second line.

```js
export default Ember.Route.extend({
  model() {
    return this.get('store').findAll('reminder');
  }
});
```

When in doubt, use a generator.

File: app/templates/reminders.hbs [hunk 6]
```diff
@@ -0,0 +1,9 @@
+	<ul>
+	{{#each model as |reminder|}}
+	<li class="spec-reminder-item">
```
Comment (stevekinney) [inline app/templates/reminders.hbs]: Is there an integration test for this?

**Pull request merged**